src/go/k8s/pkg/resources/configmap.go

```diff
@@ -282,6 +282,7 @@ func (r *ConfigMapResource) createConfiguration(
 	cr.Other["auto_create_topics_enabled"] = r.pandaCluster.Spec.Configuration.AutoCreateTopics
 	cr.Other["enable_idempotence"] = true
 	cr.Other["enable_transactions"] = true
+	cr.Other["cloud_storage_segment_max_upload_interval_sec"] = 60000 // 60s * 10 = 10 minutes
```

Comment:
I think 10 mins is aggressive by default. can we make it 30mins by default with per cloud customer override w/ whatever they want. in the case of this main customer, it should be 10mins

Reply:
@sameers361  + @vsaraswat

Reply:
10 minutes was @rkruze recommendation after consulting with the shadow indexing team. I am ok to increase it to something less aggressive if we have concerns.

Reply:
we shoul dhave input from the cloud team on how to keep configs that override the defaults